docs/operator/deploy.md

```diff
@@ -138,18 +141,30 @@ drand start --tls-disable
 
 ### Test the connection to a node
 
-Use `drand util check <address>` to test the gRPC endpoint a drand node.
+Use `drand util check <address>` to test the gRPC endpoint on a drand node.
+
+```bash
+drand util check example.com --id <beacon-id>
+
+> drand: id example.com answers correctly
+```
+
+If the address used is a DNS name, this command will try to resolve the DNS name to IP.
+
+### Test the connection to a network
```

Comment:
Can we add the command to check WITHOUT beacon id ? Just to see if the daemon is up

Reply:
The lines above it should show that scenario. I have fixed it.